tests/functional/modules/test_zos_mvs_raw_func.py

```diff
@@ -976,6 +976,52 @@ def test_data_set_name_special_characters(ansible_zos_module):
         if idcams_dataset:
             hosts.all.zos_data_set(name=idcams_dataset, state="absent")
 
+@pytest.mark.parametrize("max_rc", [2, 4, 8, 16])
+def test_new_disposition_for_data_set_members_max_rc(ansible_zos_module, max_rc):
+    idcams_dataset = None
+    try:
+        hosts = ansible_zos_module
+        default_data_set = get_tmp_ds_name()
+        default_data_set_with_member = default_data_set + '(MEM)'
+        hosts.all.zos_data_set(name=default_data_set, state="absent")
+        idcams_dataset, idcams_listcat_dataset_cmd = get_temp_idcams_dataset(hosts)
+
+        results = hosts.all.zos_mvs_raw(
+            program_name="idcams",
+            auth=True,
+            max_rc=max_rc,
+            dds=[
+                {
+                    "dd_data_set":{
+                        "dd_name":SYSPRINT_DD,
+                        "data_set_name":default_data_set_with_member,
+                        "disposition":"new",
+                        "type":"pds",
+                        "directory_blocks":15,
+                        "return_content":{
+                            "type":"text"
+                        },
+                    },
+                },
+                {
+                    "dd_input":{
+                        "dd_name":SYSIN_DD,
+                        "content":idcams_listcat_dataset_cmd
+                    }
+                },
+            ],
+        )
+        for result in results.contacted.values():
+            assert result.get("changed") is False
+            assert result.get("ret_code", {}).get("code", -1) == 8
```

Comment:
What happens with the return structure when a program returns rc = 8 and we do not fail ?

Reply:
I mean, how does it look like ?